src/plugins/intel_cpu/src/ngraph_transformations/convert_fq_rnn_to_quantized_rnn.cpp

```diff
@@ -0,0 +1,226 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "convert_fq_rnn_to_quantized_rnn.hpp"
+
+#include <cstdlib>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include <ngraph/rt_info.hpp>
+#include "ie_common.h"
+#include "ngraph/except.hpp"
+#include "ngraph/node_output.hpp"
+#include "ngraph/op/constant.hpp"
+#include "ngraph/op/convert.hpp"
+#include "ngraph/op/fake_quantize.hpp"
+#include "ngraph/op/lstm_cell.hpp"
+#include "itt.hpp"
+#include "ngraph/op/multiply.hpp"
+#include "ngraph/op/parameter.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include <ngraph_ops/type_relaxed.hpp>
+#include <stdexcept>
+#include <vector>
+
+ov::intel_cpu::ConvertFqRnnToQuantizedRnn::ConvertFqRnnToQuantizedRnn() {
+    MATCHER_SCOPE(ConvertFqRnnToQuantizedRnn);
+
+    auto X_m = ngraph::pattern::any_input();
+    auto convert_X = ngraph::pattern::wrap_type<ngraph::op::Convert>({X_m});
```

Comment:
Please use opset1::Convert instead of direct op::Convert operation. The same for all operations

Reply:
Refactored with using opset9